### PR TITLE
EDGECLOUD-2902 backend is now ComputerVision

### PR DESCRIPTION
**Pull request by brucearmstrong**
- Change findCloudlet appName parameter from "MobiledgeX SDK Demo" to "ComputerVision". Done in demo app and workshop apps.
- Fix for out of order ports in app definition: hardcode 8008 for now.
- Fixed broken unit tests. Added another.
- Misc updates for com.mobiledgex:matchingengine:2.1.2

### CODE REVIEW

File: android/MobiledgeXSDKDemo/build.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
-project.ext.matchingengineVersion = "2.1.0"
```
Comment (brucearmstrong) [inline android/MobiledgeXSDKDemo/build.gradle:3]: Version name and repo directory name are temporary until final 2.1.1 is published to maven-releases.

File: android/MobiledgeXSDKDemo/computervision/src/androidTest/java/com/mobiledgex/computervision/ComputerVisionRestUnitTest.java [hunk 2]
```diff
@@ -270,65 +281,109 @@ public void testRecognizerPredictEndpoint() {
...
 
-        String detectorDetectEnpoint = "/recognizer/train/";
-        url += detectorDetectEnpoint;
+        String detectorDetectEndpoint = "/trainer/remove/";
+        url += detectorDetectEndpoint;
         Log.i(TAG, "url is " + url);
 
-        sendPostRequest(ctx, url, null);
+        Map<String, String> params = new HashMap<>();
+        params.put("subject", "Goober");
```
Comment (lgarner) [inline android/MobiledgeXSDKDemo/computervision/src/androidTest/java/com/mobiledgex/computervision/ComputerVisionRestUnitTest.java:346]: Goober?

Comment (brucearmstrong) [reply android/MobiledgeXSDKDemo/computervision/src/androidTest/java/com/mobiledgex/computervision/ComputerVisionRestUnitTest.java:346]: https://en.wikipedia.org/wiki/Goober_Pyle

File: android/WorkshopCompleted/.idea/dictionaries/brucearmstrong.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="brucearmstrong">
+    <words>
+      <w>mobiledge</w>
```
Comment (ah1053) [inline android/WorkshopCompleted/.idea/dictionaries/brucearmstrong.xml:4]: mobiledgex

**Pull request merged**